### ISSUE TITLE
CE Table

### ISSUE THREAD

**Issue by rezayas** [HPM573/HPM573_SupportLib#14]
[**Pull first**]

In the cost-effectiveness table, the columns E[Cost] and E[Effect] should show the average cost of each strategies but it current shows the _incremental_ cost and effect with respect to the first strategy.

**Comment by yanqi8** [comment]
Updated, now only the plot function uses shifted data and all others use  original data.

**Comment by rezayas** [comment]
[**Pull first**]

Thank you! The next (and hopefully final) step is to allow the user to ask for the confidence or projection intervals be included in the cost-effectiveness table. I've added a new `Enum` to the `EconEvalClasses` and modify the definition of `build_CE_table` method so that it takes the interval type and alpha as an argument. 

When populating CE table with both estimates and confidence/prediction intervals, you might find these functions useful: `format_interval` and `format_estimate_interval` which are both in `FormatFunctions.py` file.

**Comment by yanqi8** [comment]
OK. Just to clarify, as we currently just have CI and PI for ICER, do we also want intervals for the E[cost] and E[effect] column when reporting intervals?

**Comment by rezayas** [comment]
That is correct. Every estimate should be accompanied by confidence or prediction intervals as specified by the user (or no interval at all)

**Comment by yanqi8** [comment]
Done!
